unity/XRTProject/Assets/Scripts/Networking/ApiRequest.cs

```diff
@@ -14,7 +14,7 @@ public class ApiRequest
     private string payload = string.Empty;
     private Action<UnityWebRequest> responseAction = null;
     private Action<BackendErrorResponse> errorAction = null;
-    private Action<string> responsePayload = null;
+    private Action<object> responsePayload = null;
```

Comment:
Prior to this change, all response payloads were returning `string`. Changed to `object` to allow for audio content as well.